pybamm/models/full_battery_models/lithium_ion/electrode_soh_half_cell.py

```diff
@@ -129,9 +129,7 @@ def get_initial_stoichiometry_half_cell(
     return x
 
 
-def get_min_max_stoichiometries(
-    parameter_values, options={"working electrode": "positive"}
-):
+def get_min_max_stoichiometries(parameter_values, options=None):
```

Comment:
I am not sure I understand how this change came up (or which of the bugbear rules was not happy here). Also, if we have to keep `options` as an argument with no default value, would it make sense to add it to the docstring – where it it isn't present yet?

Reply:
It is the [mutable-argument-default (B006)](https://docs.astral.sh/ruff/rules/mutable-argument-default/), basically to avoid unexpected behaviour because the same mutable object would be shared across all calls to the function.
Yep, I've now added `options` to the docstring.